scripts/mkcloud

```diff
@@ -774,9 +776,59 @@ function testsetup()
     onadmin testsetup
     local ret=$?
     $scp root@$adminip:tempest.log "$artifacts_dir/tempest.log"
+    $scp root@$adminip:tempest.subunit.log "$artifacts_dir/tempest.subunit.log"
```

Comment:
this line can be merged with the one above as $scp root@$adminip:tempest*.log "$artifacts_dir/"